courses/machine_learning/production_ml_systems/1_training_at_scale.ipynb

```diff
@@ -0,0 +1,702 @@
+{
```

Comment:
<p>You are installing the module and then running just task.py. I would suggest using python -m to run it as a module.  This way:</p><p> </p><p><code>FROM gcr.io/deeplearning-platform-release/tf2-cpu</code></p><p> </p><p><code>#RUN python3 -m pip install --upgrade --quiet tf-nightly-2.0-preview</code></p><p><code>RUN python3 -m pip install --upgrade --quiet cloudml-hypertune</code></p><p> </p><p><code>COPY . /flights</code></p><p><code>ENV PYTHONPATH ${PYTHONPATH}:/flights</code></p><p><code>ENTRYPOINT ["python3", "-m", "trainer.task"]</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/584/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Done!</p>